### DISCUSSION TITLE
several ctags for one client

### DISCUSSION THREAD

**Discussion by YBS-PC** [AdguardTeam/AdGuardHome#5888] [Q&A]
If I want to block a domain example.net for a client with ctag=device_phone AND ctag=user_child, and this domain is in the general whitelist, is the filter syntax correct 
||example.net^$important,ctag=user_child,ctag=device_phone
?
 ctag=device_phone (AND - not or) ctag=user_child,

**Comment by ameshkov** [comment]
As I recall, you should list them this way:
`||example.com^$ctag=device_phone|user_child`

`|` means `OR`.

There's no way to have `AND`, only `OR`.

More info here:
https://adguard-dns.io/kb/general/dns-filtering-syntax/#ctag

**Comment by YBS-PC** [comment]
I thought ctag was the same modifier. and the wiki says "You may want to use several modifiers in the rule. Separate the topic with commas "
